luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialPark.java

```diff
@@ -204,6 +233,7 @@ public ResourceDictionaries getDictionaries() {
         private Map<String, Factory<SearchProvider>> searchProviderFactories;
         private Map<String, Factory<KeyValueStore>> keyValueStoreFactories;
         private Map<String, Factory<ConfigPhysicalTable>> physicalTableFactories;
+        private Map<ConceptType<?>, Map<String, ? extends Factory<? extends Object>>> conceptFactoryMap;
```

Comment:
After careful considerations, I think we should not do a "poor-man's type def" even if it does seem to simplify the problem up-front. 
[The pseudo-typedef antipattern.pdf](https://github.com/yahoo/fili/files/3419511/Java.theory.and.practice_.The.pseudo-typedef.antipattern.pdf)

Reply:
The example that PDF is using is for a fairly general type that could be used throughout the program, in which case interchangeability does become a serious issue. This is a *very* specific construct used for a private variable in the LuthierIndustrialPark. I'm not really convinced the concerns raised in that PDF apply. That being said, I have no idea if subclassing a type with ? does what we would want it to do. I wouldn't be surprised if it doesn't work because of weird and subtle rules around ?.